core/runner.py

```diff
@@ -282,6 +285,29 @@ def _communicate(self, job, state):
         )
         return results
 
+    def _post_statistics(self, job):
+        """
+        Posts statistics about a training job to a cloud server
+        """
+        assert job.statistics, "No statistics to be served"
+        server_url = "http://status-server.jn6tkty4uh.us-west-1.elasticbeanstalk.com/jobs/{0}/{1}".format(job.uuid, job.dataset_uuid)
```

Comment:
I think it would be better to move to this to ServerJob, the Runner shouldn't keep track of information regarding the server. 

Within ServerJob, you could honestly just have a global variable (but it would be best to pull from config)

Reply:
I was considering that but this info will change based on the config and there's no way to change the jobs based on the config-? I put it in runner bc runner can actually get info from config

Reply:
Sure, so can you move the url to the config and have Runner pull from the config? This would probably be a secret section.

Reply:
Done